ui/app/src/components/Header/SearchBar/SearchList.tsx

```diff
@@ -38,7 +35,7 @@ function buildBoxSearchKey(resource: Resource): string {
 
 function buildRouting(resource: Resource): string {
   return isProjectMetadata(resource.metadata)
-    ? `/projects/${resource.metadata.project}/${resource.kind.toLowerCase()}s/${resource.metadata.name}`
+    ? `${ProjectRoute}/${resource.metadata.project}/${resource.kind.toLowerCase()}s/${resource.metadata.name}`
```

Comment:
I don't really see the value ? 🤔

Reply:
Cleaner

Reply:
meh I don't think it will be needed outside of this function. I'm not even convinced to keep this default function for building the routing as the "project" and "dashboard" are the only resources that respect this routing. (aka <kind_name>/<name>). For others there is no dedicated page to display it (like for global datasources, variables ...etc.)